apps/web-giddh/webpack.partial.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
             'process.env.isElectron': JSON.stringify(false),
             'process.env.errlyticsNeeded': JSON.stringify(false),
             'process.env.errlyticsKey': JSON.stringify(''),
-            'process.env.AppUrl': JSON.stringify('https://beta.giddh.com/'),
```

Comment:
is it required to remove the trailing slash?

Reply:
Updated